Formula/s/sdl3.rb

```diff
@@ -0,0 +1,65 @@
+class Sdl3 < Formula
+  desc "Low-level access to audio, keyboard, mouse, joystick, and graphics"
+  homepage "https://www.libsdl.org/"
+  url "https://github.com/libsdl-org/SDL/releases/download/release-3.2.0/SDL3-3.2.0.tar.gz"
+  sha256 "bf308f92c5688b1479faf5cfe24af72f3cd4ce08d0c0670d6ce55bc2ec1e9a5e"
+  license "Zlib"
+
+  livecheck do
+    url :stable
+    regex(/release[._-](\d+(?:\.\d+)+)/i)
+    strategy :github_latest
+  end
+
+  head do
+    url "https://github.com/libsdl-org/SDL.git", branch: "main"
+    depends_on "cmake" => :build
+  end
```

Comment:
```suggestion
  license "Zlib"
  head "https://github.com/libsdl-org/SDL.git", branch: "main"

  livecheck do
    url :stable
    regex(/release[._-](\d+(?:\.\d+)+)/i)
    strategy :github_latest
  end
```
You don't need the `head` block.